pkg/provider/cty.go

```diff
@@ -0,0 +1,109 @@
+package provider
```

Comment:
Minor nit, missing the typical copyright headers.

Reply:
This should be fixed across all files now.